### PR TITLE
fix: drag handles display

### PR DESCRIPTION
**Pull request by frosso**
Fixes #

#### Changes proposed in this Pull Request

I noticed that on `develop` the handles are still displayed when tabbing.
For accessibility reasons, they should not be displayed/tabbable.

This is how it's behaving in `develop`:

This is how it should behave:

#### Testing instructions

- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Visit http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout
- Clicking the "delete" button on one of the enabled payment methods
- The drag handle should not be clickable/tabbable

-------------------

- [ ] Added changelog entry (or does not apply)
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

### CODE REVIEW

**Comment by harriswong** [comment]
👀

File: client/components/orderable-list/style.scss [hunk 1]
```diff
@@ -24,7 +24,12 @@
 	}
 
 	&__drag-handle {
+		&#{&} {
```
Comment (harriswong) [inline client/components/orderable-list/style.scss:27]: What does this line do?

**Review by harriswong — APPROVED** [review]
👍  Tested deleting a payment method. Confirmed that the draggable icon is no longer showing and cannot be tab through.

File: client/components/orderable-list/style.scss (hunk 1 above)
Comment (frosso) [reply client/components/orderable-list/style.scss:27]: It basically repeats the class name in the compiled CSS to increase the specificity of the rule without having to use `!important`

**Pull request merged**